graylog2-server/src/main/java/org/graylog/security/authzroles/AuthzRolesResource.java

```diff
@@ -126,7 +180,53 @@ public AuthzRoleDTO get(@ApiParam(name = "roleId") @PathParam("roleId") @NotBlan
             throw new BadRequestException("Invalid argument in search query: " + e.getMessage());
         }
 
-        final PaginatedList<AuthzRoleDTO> result = authzRolesService.findPaginatedForUser(searchQuery, page, perPage,sort, order, username);
+        final PaginatedList<AuthzRoleDTO> result = authzRolesService.findPaginatedForUser(
+                searchQuery, page, perPage,sort, order, username);
         return PaginatedResponse.create("roles", result, query);
     }
+
+    @PUT
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation("Add a user to a role")
+    @AuditEvent(type = AuditEventTypes.USER_UPDATE)
+    @Path("{roleId}/assignee/add/{username}")
+    public Response addUser(
+            @ApiParam(name = "roleId") @PathParam("roleId") @NotBlank String roleId,
+            @ApiParam(name = "username") @PathParam("username") @NotBlank String username) throws ValidationException {
+        checkPermission(USERS_EDIT, username);
+
+        final User user = userService.load(username);
+        if (user == null) {
+            throw new NotFoundException("Cannot find user with name: " + username);
+        }
+        authzRolesService.get(roleId).orElseThrow(() -> new NotFoundException("Cannot find role with id: " + roleId));
+        Set<String> roles = user.getRoleIds();
+        roles.add(roleId);
+        user.setRoleIds(roles);
+        userService.save(user);
+
+        return Response.ok().build();
```

Comment:
I think we should return a `204` if we don't want to return anything. (or make the method `void` instead of returning `Response`, that's the same)
```suggestion
        return Response.noContent().build();
```